apps/create-vesting/tailwind.config.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-env node */
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+/** @type {import("tailwindcss").Config} */
+module.exports = {
+  content: [
+    "./src/**/*.{js,ts,jsx,tsx}",
+    "./node_modules/@bleu/ui/dist/**/*",
```

Comment:
```suggestion
    "./node_modules/@bleu/ui/dist/**/*",
    "../..//node_modules/@bleu/ui/dist/**/*",
```

This should make bleu/ui styling work 😉